aboutus.html

```diff
@@ -0,0 +1,76 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>KINOSTREAM.COM's Developers' Profile</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="aboutusstyle.css">
+  <link href="https://fonts.googleapis.com/css?family=Muli%7CRoboto:400,300,500,700,900" rel="stylesheet"></head>
+  <body>
+
+    <div class="main-nav">
+        <ul class="nav">
+          <li class="name"><a href="index.html">HOME | SEARCH</a></li>
+          <li><a href="signup.html">MY WATCHLIST &nbsp &nbsp|</a></li>
+          <li>ABOUT US</li>
+        </ul>
+    </div>
+    <br>
+    <div id="userNameDisplay">
+      <div id="icons"> </div>
+      <div id="icons2"> </div>
+      <div id="icons3"> </div>
+      <p id='username' style="margin-left: 10px;"> </p>
+    </div>
+
+    <header>
+      <h1 class="tag name">Hello! I’m Michael.</h1>
+      <h1 class="tag name">Hi, I’m Nick.</h1>
+      <h1 class="tag name">Hey, I’m Nurbek.</h1>
+
+    </header>
+<main>
+    <section id ='images'>
+
+      <img src="michael.jpg" alt="Picture of Michael" class="profile-image">
+      <img src="nick.jpg" alt="Picture of Nick" class="profile-image">
+      <img src="nurbek1.jpg" alt="Picture of Nurbek" class="profile-image">
+
+    </section>
+
+    <section class="flex">
+
+      <div class="card">
```

Comment:
There are likely too many `<div>'s` in play within the HTML. I would argue that there are many points through this doc in which a semantic tag could be used.